src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4517,6 +4517,11 @@ private ParameterSyntax ParseParameter()
                         this.GetExpectedTokenError(expected: SyntaxKind.ExclamationExclamationToken, actual: firstExclamation.Kind));
                 }
             }
+            else if (this.CurrentToken.Kind is SyntaxKind.ExclamationExclamationToken)
```

Comment:
I'm very confused. I'm on mobile so maybe I'm missing something. But this appears to be the same as the first check in the if-chain, right?  What am I missing?

Reply:
Ignore. I misread